cartoframes/data/observatory/entity.py

```diff
@@ -45,6 +55,32 @@ def __str__(self):
     def __repr__(self):
         return '{classname}({entity_id})'.format(classname=self.__class__.__name__, entity_id=self.id)
 
+    def _download(self, credentials=None):
+        credentials = _get_credentials(credentials)
+        user_dataset = credentials.username.replace('-', '_')
```

Comment:
could we have a helper somewhere for this kind of things? (I see we are doing this kind of replacements in several places)

Reply:
:+1: